lib/vehicle_factory.rb

```diff
@@ -0,0 +1,9 @@
+require './spec/spec_helper'
```

Comment:
No necessary. This file should be requiring the Vehicle class file and nothing else.